#!/usr/bin/env python
PACKAGE = 'tams_pr2_guzheng'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("offset_x", double_t, 1<<2, "translation offset w.r.t. frame", 0.015, -0.05, 0.05)
gen.add("offset_y", double_t, 1<<2, "translation offset w.r.t. frame", 0.0, -0.05, 0.05)
gen.add("offset_z", double_t, 1<<2, "translation offset w.r.t. frame", -0.01, -0.05, 0.05)

exit(gen.generate(PACKAGE, "", "Offsets"))
